code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -718,6 +718,20 @@
 						to_chat(src, "<span class='danger'>Your thinking is clouded and distant.</span>")
 					else if(oxyloss > 30)
 						to_chat(src, "<span class='danger'>You're choking!</span>")
+
+			if(nutrition > NUTRITION_LEVEL_FULL)

```

Comment:
you should be using a switch statement for this

Reply:
Still learning byond code but I think I got it